packages/visualizations/src/components/Map/utils.ts

```diff
@@ -180,5 +183,5 @@ export const getFixedTooltips = (shapeKeys, features, renderTooltip) => {
         return null;
     });
 
-    return popups;
+    return popups.filter(Boolean) as ChoroplethFixedTooltipDescription[];
```

Comment:
So apparently [Boolean doesn't have the right signature for TS](https://github.com/microsoft/TypeScript/issues/16069). I really like the [isPresent ](https://www.npmjs.com/package/ts-is-present) solution, although we could craft our own function easily too.

Whatever the solution, I think we should do the extra effort to `popus.filter<ChoroplethFixedTooltipDescription>(…)` rather than hard casting. But this bug is a bit of pain indeed…

Reply:
I spent a lot of time trying to do it the right way, but somehow didn't succeed. I'll try using `compact` from `lodash` which should do the same thing, and has types that should work here.

Reply:
If we don't want to use the npm package `ts-is-present`, the other solution seems to be using [type guard returns](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates):
```
function nonNullable<T>(value: T): value is NonNullable<T> {
  return value !== null && value !== undefined;
}

return popus.filter<ChoroplethPopups>(nonNullable)
```

Reply:
I used an inline function:
```ts
return popups.filter((item): item is NonNullable<ChoroplethFixedTooltipDescription> => Boolean(item)) as ChoroplethFixedTooltipDescription[];
```

Thank you for teaching us about `is` 😁